apps/site/pages/courses/[courseSlug]/[lessonSlug]/index.tsx

```diff
@@ -87,9 +87,13 @@ function usePreferredMediaType(lesson: LessonProps["lesson"]) {
 	// Handle situations that content creator may created an empty lesson (to add content later)
 	const content = lesson.content as LessonContent;
 	const router = useRouter();
-	const [preferredMediaType, setPreferredMediaType] = useState(
-		content.length > 0 ? content[0].type : null
-	);
+	const [preferredMediaType, setPreferredMediaType] = useState<
+		"video" | "article" | "pdf" | "iframe"
+	>("video");
```

Comment:
es gibt einen typen in `lesson-context.tsx` dafür, der aber aktuell noch nicht verfügbar ist. ggf hier refactorn. 

```ts
const contentTypes: LessonContentMediaType[] = ["video", "article", "pdf", "iframe"];
```

oder zB folgendes ableiten:

```ts
type MediaType = (typeof content)[number]['type'];
```